src/cn.js

```diff
@@ -688,7 +688,7 @@
           pass: q.ssh_pass.value,
           key: q.ssh_key.value,
         },
-        '/bin/ls -1 /opt/mdyalog/*/*/*/mapl /opt/mdyalog/*/*/*/*/mapl /Applications/Dyalog-*/Contents/Resources/Dyalog/mapl 2>/dev/null',
+        'PATH=/ setopt NO_NOMATCH 2>/dev/null ; /bin/ls -1 /opt/mdyalog/*/*/*/mapl /opt/mdyalog/*/*/*/*/mapl /Applications/Dyalog-*/Contents/Resources/Dyalog/mapl 2>/dev/null',
```

Comment:
```suggestion
        'PATH=/ unsetopt NOMATCH 2>/dev/null ; /bin/ls -1 /opt/mdyalog/*/*/*/mapl /opt/mdyalog/*/*/*/*/mapl /Applications/Dyalog-*/Contents/Resources/Dyalog/mapl 2>/dev/null',
```

Reply:
Just got around to look into this and I can't come up with a better solution, just a minor tweak to make it more readable.